angular/demo/daisyui/src/app/samples/toast/toast.component.ts

```diff
@@ -0,0 +1,37 @@
+import {BaseWidgetDirective, type ToastWidget, auBooleanAttribute, callWidgetFactory, createToast} from '@agnos-ui/angular-headless';
+import {ChangeDetectionStrategy, Component, EventEmitter, Input, Output} from '@angular/core';
+
+@Component({
+	selector: 'app-toast',
+	template: `
+		@if (!state().hidden) {
+			<div class="alert {{ state().className }}">
+				<ng-content />
+				@if (state().dismissible) {
+					<button class="btn btn-sm btn-circle btn-ghost justify-self-end" (click)="api.close()" [attr.aria-label]="state().ariaCloseButtonLabel">
+						✕
```

Comment:
not sure, but I would prefer to put an svg (that's what I am doing in the alert pr)

Reply:
not a big fun of svg here, it just take space in the demo page

Reply:
The svg looks better, so I prefer the svg too.
For the concern of svg taking place, you can put it in the common folder.